.gitignore

```diff
@@ -111,6 +111,12 @@ fabric.properties
 .idea/**/markdown-navigator.xml
 .idea/**/markdown-navigator/
 
+# GoLand files
+.idea/.gitignore
+.idea/kube-job-notifier.iml
+.idea/modules.xml
+.idea/vcs.xml
```

Comment:
I don't think stuff specific to your own environment belong in this file, normally you can ignore these kind of files in a global gitignore!